### PR TITLE
Describe the delegation threshold

### PR DESCRIPTION
**Pull request by sgkim126**

### CODE REVIEW

File: spec/Dynamic-Validator.md [hunk 1]
```diff
@@ -67,7 +67,16 @@ The election is a process that elects validators of a term according to the foll
 
 1. Pick **MAX_NUM_OF_VALIDATORS** candidates in order of having received many delegations.
 2. Select **MIN_NUM_OF_VALIDATORS** accounts; they become validators.
-3. Among the rest of them, drop the accounts having received less than MIN_CCS_RATE_TO_BE_VALIDATOR; the remains become validators.
+3. Among the rest of them, drop the accounts having received less than **DELEGATION_THRESHOLD**; the remains become validators.
```
Comment (ScarletBlue) [inline spec/Dynamic-Validator.md]: Among the rest of them, drop the accounts that received less than **DELEGATION_THRESHOLD**; the remaining accounts become validators.

File: spec/Dynamic-Validator.md [hunk 2]
```diff
@@ -67,7 +67,16 @@ The election is a process that elects validators of a term according to the foll
...
+
+This process guarantes two things.
```
Comment (ScarletBlue) [inline spec/Dynamic-Validator.md]: This process guarantees two things:

File: spec/Dynamic-Validator.md [hunk 3]
```diff
@@ -67,7 +67,16 @@ The election is a process that elects validators of a term according to the foll
...
+* There are **MIN_NUM_OF_VALIDATORS** validators if the number of candidates are larger than **MIN_NUM_OF_VALIDATORS**.
+* The candidates that are not in **MIN_NUM_OF_VALIDATORS** and not receiving delegation more than **DELEGATION_THRESHOLD** will not be validators.
```
Comment (ScarletBlue) [inline spec/Dynamic-Validator.md]: * The candidates that are not in **MIN_NUM_OF_VALIDATORS** and not receiving delegation of more than **DELEGATION_THRESHOLD** will not be validators.

File: spec/Dynamic-Validator.md [hunk 4]
```diff
@@ -67,7 +67,16 @@ The election is a process that elects validators of a term according to the foll
...
+
+### Delegation Threshold
+It's a constant threshold to prevent the account from being a validator with little delegations.
+Current **DELEGATION_THRESHOLD** is `100,000`, 1% of total CCS(`10,000,000`).
```
Comment (ScarletBlue) [inline spec/Dynamic-Validator.md]: Current **DELEGATION_THRESHOLD** is `100,000`, which is 1% of the total CCS(`10,000,000`).

File: spec/Dynamic-Validator.md [hunk 5]
```diff
@@ -67,7 +67,16 @@ The election is a process that elects validators of a term according to the foll
...
+It means there can be 100 valid candidates who can be validators.
```
Comment (ScarletBlue) [inline spec/Dynamic-Validator.md]: It means there can be 100 valid candidates with the potential to be validators.

File: spec/Dynamic-Validator.md [hunk 6]
```diff
@@ -67,7 +67,16 @@ The election is a process that elects validators of a term according to the foll
 
 1. Pick **MAX_NUM_OF_VALIDATORS** candidates in order of having received many delegations.
 2. Select **MIN_NUM_OF_VALIDATORS** accounts; they become validators.
-3. Among the rest of them, drop the accounts having received less than MIN_CCS_RATE_TO_BE_VALIDATOR; the remains become validators.
+3. Among the rest of them, drop the accounts that received less than **DELEGATION_THRESHOLD**; the remaining accounts become validators.
+
+This process guarantees two things:
+* There are **MIN_NUM_OF_VALIDATORS** validators if the number of candidates are larger than **MIN_NUM_OF_VALIDATORS**.
```
Comment (kseo) [inline spec/Dynamic-Validator.md]: Isn't it MAX_NUM_OF_VALIDATORS?

Comment (sgkim126) [reply spec/Dynamic-Validator.md]: No. It isn't. **MIN_NUM_OF_VALIDATORS** is correct.
In the current constants, CodeChain guarantees 4 validators only when there are 4 valid candidates.

Comment (sgkim126) [reply spec/Dynamic-Validator.md]: I changed the statement to remove the ambiguity.

File: spec/Dynamic-Validator.md [hunk 7]
```diff
@@ -67,7 +67,16 @@ The election is a process that elects validators of a term according to the foll
...
 2. Select **MIN_NUM_OF_VALIDATORS** accounts; they become validators.
-3. Among the rest of them, drop the accounts having received less than MIN_CCS_RATE_TO_BE_VALIDATOR; the remains become validators.
+3. Among the rest of them, drop the accounts that received less than **DELEGATION_THRESHOLD**; the remaining accounts become validators.
+
+This process guarantees two things:
+* There are at least **MIN_NUM_OF_VALIDATORS** validators only if the number of candidates is larger than **MIN_NUM_OF_VALIDATORS**.
+* The candidates that are not in **MIN_NUM_OF_VALIDATORS** and not receiving delegation of more than **DELEGATION_THRESHOLD** will not be validators.
+
+### Delegation Threshold
+It's a constant threshold to prevent the account from being a validator with little delegations.
```
Comment (ScarletBlue) [inline spec/Dynamic-Validator.md]: It's a constant threshold to prevent the accounts with little delegations from becoming validators.

**Pull request merged**